<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="HEU-PEDRO-PRADIITA" failures="8" tests="27" name="RegisterPage" time="187.951" errors="0" timestamp="22 Oct 2021 04:13:39 GMT">
  <testcase classname="brick.BrickRegisterTest" name="ASuccessRegist" time="1.515"/>
  <testcase classname="brick.BrickRegisterTest" name="AccessApiText" time="0.048"/>
  <testcase classname="brick.BrickRegisterTest" name="AddressText" time="0.04">
    <failure type="java.lang.AssertionError" message="expected [Address] but found [Your Address]">
      <![CDATA[java.lang.AssertionError: expected [Address] but found [Your Address]
	at brick.BrickRegisterTest.AddressText(BrickRegisterTest.java:287)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
	at java.base/java.lang.Thread.run(Thread.java:831)
... Removed 21 stack frames]]>
    </failure>
  </testcase> <!-- AddressText -->
  <testcase classname="brick.BrickRegisterTest" name="AlreadyAccountText" time="0.05"/>
  <testcase classname="brick.BrickRegisterTest" name="BrickSdkText" time="0.039"/>
  <testcase classname="brick.BrickRegisterTest" name="ConfirmPasswordAlert" time="0.122">
    <failure type="java.lang.AssertionError" message="expected [Please enter a password.] but found [Please enter a password]">
      <![CDATA[java.lang.AssertionError: expected [Please enter a password.] but found [Please enter a password]
	at brick.BrickRegisterTest.ConfirmPasswordAlert(BrickRegisterTest.java:178)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
	at java.base/java.lang.Thread.run(Thread.java:831)
... Removed 21 stack frames]]>
    </failure>
  </testcase> <!-- ConfirmPasswordAlert -->
  <testcase classname="brick.BrickRegisterTest" name="ConfirmPasswordText" time="0.038"/>
  <testcase classname="brick.BrickRegisterTest" name="EmailAlert" time="0.125">
    <failure type="java.lang.AssertionError" message="expected [Please provide an email.] but found [Please provide an email]">
      <![CDATA[java.lang.AssertionError: expected [Please provide an email.] but found [Please provide an email]
	at brick.BrickRegisterTest.EmailAlert(BrickRegisterTest.java:90)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
	at java.base/java.lang.Thread.run(Thread.java:831)
... Removed 21 stack frames]]>
    </failure>
  </testcase> <!-- EmailAlert -->
  <testcase classname="brick.BrickRegisterTest" name="EmailSales" time="0.103"/>
  <testcase classname="brick.BrickRegisterTest" name="EmailText" time="0.045">
    <failure type="java.lang.AssertionError" message="expected [Email] but found [Your Email]">
      <![CDATA[java.lang.AssertionError: expected [Email] but found [Your Email]
	at brick.BrickRegisterTest.EmailText(BrickRegisterTest.java:257)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
	at java.base/java.lang.Thread.run(Thread.java:831)
... Removed 21 stack frames]]>
    </failure>
  </testcase> <!-- EmailText -->
  <testcase classname="brick.BrickRegisterTest" name="FirstNameText" time="0.039"/>
  <testcase classname="brick.BrickRegisterTest" name="FreeDeveloperText" time="0.037"/>
  <testcase classname="brick.BrickRegisterTest" name="FullNameAlert" time="0.127">
    <failure type="java.lang.AssertionError" message="expected [Please enter a firstname.] but found [Please enter a firstname]">
      <![CDATA[java.lang.AssertionError: expected [Please enter a firstname.] but found [Please enter a firstname]
	at brick.BrickRegisterTest.FullNameAlert(BrickRegisterTest.java:56)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
	at java.base/java.lang.Thread.run(Thread.java:831)
... Removed 21 stack frames]]>
    </failure>
  </testcase> <!-- FullNameAlert -->
  <testcase classname="brick.BrickRegisterTest" name="IntegrateApiText" time="0.036"/>
  <testcase classname="brick.BrickRegisterTest" name="InvalidConfirmPasswordAlert" time="0.253"/>
  <testcase classname="brick.BrickRegisterTest" name="InvalidEmailAlert" time="0.182"/>
  <testcase classname="brick.BrickRegisterTest" name="InvalidPasswordAlert" time="0.19"/>
  <testcase classname="brick.BrickRegisterTest" name="LastNameAlert" time="0.117">
    <failure type="java.lang.AssertionError" message="expected [Please enter a lastname.] but found [Please enter a lastname]">
      <![CDATA[java.lang.AssertionError: expected [Please enter a lastname.] but found [Please enter a lastname]
	at brick.BrickRegisterTest.LastNameAlert(BrickRegisterTest.java:73)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
	at java.base/java.lang.Thread.run(Thread.java:831)
... Removed 21 stack frames]]>
    </failure>
  </testcase> <!-- LastNameAlert -->
  <testcase classname="brick.BrickRegisterTest" name="LastNameText" time="0.036"/>
  <testcase classname="brick.BrickRegisterTest" name="LoginTextLink" time="2.092"/>
  <testcase classname="brick.BrickRegisterTest" name="LogoBrick" time="0.033"/>
  <testcase classname="brick.BrickRegisterTest" name="PasswordAlert" time="0.119">
    <failure type="java.lang.AssertionError" message="expected [Please enter a password.] but found [Please enter a password]">
      <![CDATA[java.lang.AssertionError: expected [Please enter a password.] but found [Please enter a password]
	at brick.BrickRegisterTest.PasswordAlert(BrickRegisterTest.java:142)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
	at java.base/java.lang.Thread.run(Thread.java:831)
... Removed 21 stack frames]]>
    </failure>
  </testcase> <!-- PasswordAlert -->
  <testcase classname="brick.BrickRegisterTest" name="PasswordText" time="0.038"/>
  <testcase classname="brick.BrickRegisterTest" name="PhoneAlert" time="0.389"/>
  <testcase classname="brick.BrickRegisterTest" name="PhoneNumberText" time="0.036">
    <failure type="java.lang.AssertionError" message="expected [Phone Number] but found [Your Phone Number]">
      <![CDATA[java.lang.AssertionError: expected [Phone Number] but found [Your Phone Number]
	at brick.BrickRegisterTest.PhoneNumberText(BrickRegisterTest.java:272)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
	at java.base/java.lang.Thread.run(Thread.java:831)
... Removed 21 stack frames]]>
    </failure>
  </testcase> <!-- PhoneNumberText -->
  <testcase classname="brick.BrickRegisterTest" name="QuestiontText" time="0.036"/>
  <testcase classname="brick.BrickRegisterTest" name="SignUpText" time="0.038"/>
</testsuite> <!-- RegisterPage -->
